packages/devtools_app/lib/src/shared/primitives/custom_pointer_scroll_view.dart

```diff
@@ -812,9 +812,7 @@ class _RenderScrollSemantics extends RenderProxyBox {
     }
 
     _innerNode ??= SemanticsNode(showOnScreen: showOnScreen);
-    _innerNode!
-      ..isMergedIntoParent = node.isPartOfNodeMerging
-      ..rect = Offset.zero & node.rect.size;
+    _innerNode!.rect = Offset.zero & node.rect.size;
```

Comment:
can we do instead what is done on flutter master?
`(_innerNode ??= SemanticsNode(showOnScreen: showOnScreen)).rect = node.rect;`

(Most of the code in this file is copied from `scrollable.dart` from the Flutter framework)